### PR TITLE
Create root project directory

### LINKED ISSUE #24: Shuffle folders around
Right now, we have:

1) /opt/app_name/, which contains a git clone
2) /opt/venv_app_name/, which contains its venv
3) /opt/app_name/gunicorn_config.py, inside the git clone dir
4) settings need to be put somewhere, currently, the logical place is inside the git clone

We should give these a dir for themselves, something like this:

```
/opt/app_name
    /source
    /venv
```

Inside there, you'd be free to put config files -- this is where gunicorn_config.py would go, and where What's Fresh's config/config.yml would go.

### PR DESCRIPTION
**Pull request by tschuy**
:musical_note: It's time to do the directory shuffle!

Fixes #24 . Shuffles directories around so that gunicorn_config and /app/config/config.yml can live happily in peace without being in a git repo. Tree now looks like this:

```
/opt/my_app:
         /source/
         /venv/
```

the gunicorn_config file is generated in /opt/my_app.

### CODE REVIEW

**Comment by tschuy** [comment]
I have the integration tests running in the background, but have to head out for the day. If someone else wants to run these ( @iankronquist I know you're anxious to get this whole thing released) you can, or I can just check on the outcome tomorrow.

File: spec/unit/default_spec.rb [hunk 1]
```diff
@@ -123,13 +123,14 @@
...
+    expect(chef_run).to create_python_virtualenv('/opt/tutorial_c/venv').with(
       owner: 'chef', group: 'chef')
   end
 
   it 'installs special_requirements' do
     expect(chef_run)
-      .to install_python_pip('/opt/tutorial_c/special_requirements.txt').with(
+      .to install_python_pip('/opt/tutorial_c/source/' \
+        'special_requirements.txt').with(
         options: '-r')
```
Comment (jordane) [inline spec/unit/default_spec.rb]: This needs to be indented. Check travis for details.

**Comment by iankronquist** [comment]
Whoops, I accidentally clicked the close button.
In any case, the integration tests passed. Everything looks good to me.

**Comment by jordane** [comment]
@tschuy @iankronquist mind squashing the indentation error commit please?

**Comment by iankronquist** [comment]
@jordane Sqvashed.

**Comment by jordane** [comment]
@iankronquist @tschuy Next time please wait until I actually +1 this. I wanted to check the squashed commit to make sure it was done correctly..

**Pull request merged**